distributed/src/main/scala/asobu/distributed/EndpointsRegistry.scala

```diff
@@ -0,0 +1,139 @@
+package asobu.distributed
+
+import akka.actor._
+import akka.cluster.{Member, MemberStatus, Cluster}
+import akka.cluster.ddata.{LWWMap, LWWMapKey, DistributedData}
+import akka.cluster.ddata.Replicator._
+import asobu.distributed.EndpointsRegistry.DocDataType
+import play.api.libs.json.{JsNumber, Json, JsObject}
+import concurrent.duration._
+
+trait EndpointsRegistry {
+  val EndpointsDataKey = EndpointsRegistry.EndpointsDataKey
+  val EndpointsDocsKey = EndpointsRegistry.EndpointsDocsKey
+
+  def writeConsistency: WriteConsistency
+  def readConsistency: ReadConsistency
+  def emptyData: LWWMap[EndpointDefinition]
+  def emptyDocs: LWWMap[DocDataType]
+
+  implicit def node: Cluster
+  def replicator: ActorRef
+}
+
+object EndpointsRegistry {
+
+  //Doc json is stored as String (JsObject isn't that serializable after manipulation)
+  type DocDataType = String
+
+  val EndpointsDataKey = LWWMapKey[EndpointDefinition]("endpoints-registry-endpoints")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name 'EndpointsDataKey' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/iHeartRadio/asobu/file/2323047312/issues/source?bid=3037259&fileBranchId=3178877#l29)